main.py

```diff
@@ -89,6 +97,20 @@ def search_key():
                 elif startRect.collidepoint(event.pos):
                     start = True
                     break
+                
+                elif incTurnsRect.collidepoint(event.pos):
+                    if maxturns<8:
+                        maxturns=maxturns+1
+                        
+                    elif maxturns>8:
+                        maxturns = maxturns+0
+                        
+                elif decTurnsRect.collidepoint(event.pos):
+                    if maxturns>4:
+                        maxturns=maxturns-1
+
+                    elif maxturns<=4:

```

Comment:
Same here, no need of this because it essentially means you're not changing anything:

```
                    elif maxturns<=4:
                        maxturns = maxturns-0
```